docs/pages/application-access/guides/usage.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: Using Application Access Guide
+description: How to configure Teleport for Application Access
+---
+
+# Application Access Usage
+
+Download the latest version of Teleport for your platform from our [downloads page](https://goteleport.com/teleport/download)
+and follow the installation [instructions](../../installation.mdx).
+
+## Start Auth/Proxy Service
+
+Create a configuration file for a Teleport service that will be running
+auth and proxy servers:
+
+```yaml
+teleport:
+  data_dir: /var/lib/teleport
+auth_service:
+  enabled: "yes"
+proxy_service:
+  enabled: "yes"
+  # Set public address proxy will be reachable at.
+  public_addr: teleport.example.com:3080
+ssh_service:
+  enabled: "no"
+```
+
+Start the service:
+
+```sh
+$ teleport start --config=/path/to/teleport.yaml
+```
+
+### Generate Token
+
+A join token is required to authorize a Teleport Application Proxy service to
+join the cluster. Generate a short-lived join token and save it for example
+in `/tmp/token`:
+
+```sh
+$ tctl tokens add \
+    --type=app \
+    --app-name=grafana \
+    --app-uri=http://localhost:3000
+```
+
+### TLS Requirements
+
+TLS is required to secure Teleport's Unified Access Plane and any connected
+applications. When setting up Teleport, the minimum requirement is a certificate
+for the proxy and a wildcard certificate for its sub-domain. This is where
+everyone will log into Teleport.
+
+In our example:
+
+* `teleport.example.com` will host the Access Plane.
+* `*.teleport.example.com` will host all of the applications e.g. `grafana.teleport.example.com`.
+
+Teleport supports accessing these applications on other domains if required.
+DNS entries must be configured and the correct certificates need to be obtained
+for this to work.
+
+```yaml
+proxy_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3023
+  tunnel_listen_addr: 0.0.0.0:3024
+  public_addr: teleport.example.com:443
+  # Example of two HTTPs keypairs.
+  https_keypairs:
+  - key_file: /etc/letsencrypt/live/teleport.example.com/privkey.pem
+    cert_file: /etc/letsencrypt/live/teleport.example.com/fullchain.pem
+  - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
+    cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
+```
+
+<Admonition
+  type="tip"
+  title="Using certbot to obtain wildcard certs"
+>
+  Let's Encrypt provides free wildcard certificates. If using [certbot](https://certbot.eff.org/)
+  with DNS challenge, the below script will make setup easy. Update [EMAIL] and [DOMAIN]
+
+  ```sh
+  certbot certonly --manual \
+    --preferred-challenges=dns \
+    --email [EMAIL] \
+    --agree-tos \
+    --manual-public-ip-logging-ok \
+    -d "teleport.example.com, *.teleport.example.com"
+  ```
+</Admonition>
+
+### Create a User
+
+In order to be able to access the application, a Teleport user needs to have
```

Comment:
I would start with who needs what here: `A teleport user needs a role's permission to access an application.